.storybook/components/CodeViewer/CodeViewer.helpers.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import reactElementToJSXString from "react-element-to-jsx-string";
+
+// modified getReactElementDisplayName from react-element-to-jsx-string
+// https://github.com/algolia/react-element-to-jsx-string/blob/master/src/parser/parseReactElement.js#L15-L20
+function getDisplayName(element) {
+  const displayName =
+    element.type.displayName ||
+    (element.type.name !== "_default" ? element.type.name : null) || // function name
+    (typeof element.type === "function" // function without a name, you should provide one
+      ? "Anonymous"
+      : element.type);
+
+  return typeof displayName === "string" ? displayName.replace(/^Svg/, "") : displayName;
+}
+
+const formatOptions = {
+  showFunctions: true,
+  showDefaultProps: false,
+  filterProps: ["key"],
+  maxInlineAttributesLineLength: 40,
+  displayName: getDisplayName,
+};
+
+export function getJSX(children) {
+  return Array.isArray(children)
+    ? children.map(child => getJSX(child)).join("\n")
+    : reactElementToJSXString(children, formatOptions);
+}
+
+export function copyToClipboard(selector) {
+  const code = document.querySelector(selector).textContent;
```

Comment:
The snippet we use in LP is shorter, I wonder if there are any downsides
https://github.com/acl-services/accounts/blob/f00ae3263350412e8fa2f4978ef1c96f4c8a9715/client/app/bundles/accounts/containers/CopyButton.js#L10-L17

Reply:
Basically the same thing – I'm restoring the focus to where it came from (might be a good thing for you guys to do too, for accessibility) and adding some CSS to move the temporary element off screen.  If you guys don't seem to need the CSS protection, I probably don't either.